metadata_backend/helpers/schemas/folders.json

```diff
@@ -22,6 +22,573 @@
+        "alternateIdentifiers": {
+          "type": "array",
+          "title": "List of alternate identifiers",
+          "items": {
+            "type": "object",
+            "title": "An identifier or identifiers other than the primary Identifier of the resource",
+            "required": [
+              "alternateIdentifier",
+              "alternateIdentifierType"
+            ],
+            "properties": {
+              "alternateIdentifier": {
+                "type": "string",
+                "title": "Alternate identifier info"
+              },
+              "alternateIdentifierType": {
+                "type": "string",
+                "title": "Type of alternate identifier"
+              }
+            },
+            "additionalProperties": false
+          },
+          "uniqueItems": true
+        },
+        "relatedIdentifiers": {
+          "type": "array",
+          "title": "List of related identifiers",
+          "items": {
+            "type": "object",
+            "title": "Identifier of related resources",
+            "required": [
+              "relatedIdentifier",
+              "relatedIdentifierType",
+              "relationType"
+            ],
+            "properties": {
+              "relatedIdentifier": {
+                "type": "string",
+                "title": "Related identifier info"
+              },
+              "relatedIdentifierType": {
+                "type": "string",
+                "title": "Type of related identifier"
+              },
+              "relationType": {
+                "type": "string",
+                "title": "Specification of the relation"
+              },
+              "relatedMetadataScheme": {
+                "type": "string",
+                "title": "Scheme of related metadata"
+              },
+              "schemeUri": {
+                "type": "string",
+                "title": "URI (location) of the related metadata scheme"
+              },
+              "schemeType": {
+                "type": "string",
+                "title": "Type of the related metadata scheme"
+              },
+              "resourceTypeGeneral": {
+                "type": "string",
+                "title": "Optional general type name"
+              }
+            },
+            "additionalProperties": false
+          },
+          "uniqueItems": true
+        },
+        "sizes": {
+          "type": "array",
+          "title": "List of sizes",
+          "items": {
+            "type": "string",
+            "title": "Unstructured size information about the resource"
+          }
+        },
+        "formats": {
+          "type": "array",
+          "title": "List of formats",
+          "items": {
+            "type": "string",
+            "title": "Technical format of the resource"
+          }
+        },
+        "version": {
+          "type": "string",
+          "title": "Version number of the resource"
+        },
+        "rightsList": {
```

Comment:
`rights` is a bit in the air, but most data is going to be under a Data Access Committee so not sure we are going to make use of this property, we can leave it out for now.

Reply:
removed